definitions/checks/puppet/verify_no_empty_cacert_requests.rb

```diff
@@ -0,0 +1,20 @@
+module Checks::Puppet
+  class VerifyNoEmptyCacertRequests < ForemanMaintain::Check
+    metadata do
+      label :check_no_empty_cacert_requests_exist
```

Comment:
I suggest having `puppet` in the prefix of the label, something like `label :puppet_check_no_empty_cert_requests`